src/lz4frame.jl

```diff
@@ -94,8 +105,8 @@ struct LZ4F_preferences_t
     reserved::NTuple{4, Cuint}        
 end
 
-function LZ4F_preferences_t(frame_info::LZ4F_frameInfo_t; compressionlevel::Cint=(Cint)(0), autoflush::Cuint=(Cuint)(0))
-    LZ4F_preferences_t(frame_info, compressionlevel, autoflush, (0,0,0,0))
+function LZ4F_preferences_t(frame_info::LZ4F_frameInfo_t; compressionlevel::Int=0, autoflush::Bool=false)
+    LZ4F_preferences_t(frame_info, Cint(compressionlevel), autoflush? Cuint(1):Cuint(0), (0,0,0,0))
```

Comment:
Ternary operators should use spaces surrounding the `?` and `:`.
```julia
autoflush ? Cuint(1) : Cuint(0)
```
I'll note you can just write this as:
```julia
Cuint(autoflush)
```